ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/versions/deneb/BlobSidecar.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Would be good to add a property test similar to `BlobsSidecarPropertyTest`

Reply:
Done